### PR TITLE
Filter licences by status on the 2PT Review Licences Page

### PR DESCRIPTION
**Pull request by Jozzey**
https://eaflood.atlassian.net/browse/WATER-4217

Add a filter to the two-part tariff Review Licences page to allow the user to filter by license status.

### CODE REVIEW

File: app/services/bill-runs/two-part-tariff/fetch-bill-run-licences.service.js
Comment (Cruikshanks) [inline app/services/bill-runs/two-part-tariff/fetch-bill-run-licences.service.js:1]: The one change I'd suggest making is where `filterData` gets put together.

We're passing in the data for it from `review-bill-run.service.js`. I reckon that's a job that is better suited to being _in_ `review-bill-run.service.js`.

If then the function signature for `fetch-bill-run-licences.service.js` is updated to `(id, licenceHolder = null, status = null)` this service becomes a bit cleaner and what the args are is clearer.

We also avoid passing the payload on further than we need to. I hope that makes sense?

Comment (Jozzey) [reply app/services/bill-runs/two-part-tariff/fetch-bill-run-licences.service.js:1]: Agreed, that would make it clearer. I'll make that change.

Comment (Jozzey) [reply app/services/bill-runs/two-part-tariff/fetch-bill-run-licences.service.js:1]: Made some changes and renamed a load of stuff. Hopefully a bit nicer now :grin:

**Review by Cruikshanks — APPROVED** [review]

**Pull request merged**